src/pages/AssetPage/assetPage.test.tsx

```diff
@@ -4,46 +4,99 @@ import { render, screen, waitForElementToBeRemoved } from '@testing-library/reac
 import { getUserAssets } from '../../api/assets-api';
 import { useWalletHook } from '../../components/NavBar/useWallethook';
 
-const assetData = {
-    result: [{
-        image_url: 'img',
-        collection: {
-            name: 'colllectionName'
-        },
-        name: 'testname',
-    }]
-}
-
-const wallet = {
-    walletInfo: {
-        address: "0xe3ece548F1DD4B1536Eb6eE188fE35350bc1aa1a"
-    }
-}
-
 jest.mock('../../api/assets-api');
 jest.mock('../../components/NavBar/useWallethook')
 
 describe('Asset page', () => {
-    beforeEach(() => {
-        (useWalletHook as jest.Mock).mockReturnValue(wallet);
-        (getUserAssets as jest.Mock).mockResolvedValue(assetData);
-    });
+    describe('when the wallet information exists', () => {
```

Comment:
this description is not correct as last it cover situation without wallet information. 

Better to have here smth related rendering AssetPage.

Reply:
I don't see anything bad here. I would say naming might be better, however it's following common naming convention (Given, When, Then) https://markus.oberlehner.net/blog/naming-your-unit-tests-it-should-vs-given-when-then/

Reply:
I have another `describe` on line 92 :)
`describe('when the wallet information does not exist', () => {`